src/components/Search/NewMenuItem/NewMenuItem.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  StyledLink,
+  Wrapper,
+  Subcopy,
+  Icon,
+  InnerContent,
+  ArrowIcon,
+} from './NewMenuItem.style';
+import { Region, MetroArea, County } from 'common/regions';
+import { formatEstimate } from 'common/utils';
+import { getLocationIconFillColor } from 'components/Search';
+import { StyledRegionName } from 'components/SharedComponents';
+
+const NewMenuItem: React.FC<{ region: Region; zipCodeInput: string }> = ({
```

Comment:
Can you add a note to remove New from the name when we retire the current MenuItem component?

Reply:
theres an item in my homepage checklist for this! will remove the old `MenuItem` and move+rename this one